knowledge_graph/network_class.py

```diff
@@ -1,4 +1,7 @@
+import typing
+import owlready2
 from owlready2 import *
```

Comment:
is owlready2 getting imported twice?

Reply:
I think `from owlready2 import *` was supposed to be deleted. I want to replace this implicit import with explicit imports. Under-the-hood, it's not actually getting imported twice. The first import would import the module and the second import adds the imported functions to the global namespace. It would be ugly, but we could actually repeat `import owlready2` as many times as we wanted after this without any additional computational overhead. It would just be nonsensical clutter.